module.js

```diff
@@ -0,0 +1,136 @@
+const catchA = function () {
+    let name = '';
```

Comment:
maybe consider making something like this into an object. It will make your code look a lot cleaner

`const game = {name: '' , start: false, score: 0, level: 1, iphone: []}`